examples/auth_oauth_jwt.py

```diff
@@ -23,14 +23,15 @@
 jwt_oauth_client_id = os.getenv("COZE_JWT_OAUTH_CLIENT_ID")
 # private key
 jwt_oauth_private_key = os.getenv("COZE_JWT_OAUTH_PRIVATE_KEY")
+# The private key file that stores the private key with .pem extension
+jwt_oauth_private_file = os.getenv("COZE_JWT_OAUTH_PRIVATE_FILE")
```

Comment:
@chyroc 
It's a bit ambiguous what should be stored in such environmental variable, so, I think, it would be better to clarify it and write something like "Path to the private key file" in this comment, so it's easily understandable that the variable stores a path string, not the file content itself etc.

Reply:
It makes sense. Can you please submit a PR to complete this change? Thank you.

Reply:
Done. Please check the new PR.